{{cookiecutter.github_repository}}/pyproject.toml

```diff
@@ -0,0 +1,147 @@
+[tool.poetry]
+name = "{{cookiecutter.github_repository}}"
+version = "{{cookiecutter.version}}"
+description = "{{cookiecutter.project_description}}"
+authors = ["{{cookiecutter.default_from_email}}"]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+Django = "^3.2.10"
+six = "^1.16.0"
+django-environ = "^0.8.1"
```

Comment:
As we started using caret notation for dependency specification, we can omit the last patch version for all dependencies following semver.

https://python-poetry.org/docs/dependency-specification/